components/Comment/lib/types.ts

```diff
@@ -35,8 +34,8 @@ export const parseComment = ({ raw, parent }: parseCommentArgs): Comment => {
     createdDate: formatDateStandard(raw.created_date),
     updatedDate: formatDateStandard(raw.created_date),
     timeAgo: timeSince(raw.created_date),
-    createdBy: parseCreatedBy(raw.created_by),
-    content: raw.content,
+    createdBy: parseUser(raw.created_by),
```

Comment:
@lightninglu10 Just FYI, I  changed the name of this mainly because I think I picked a wrong name.
There are instances where I need to pass in the currently logged in user and its a bit awkward to do `parseCreatedBy(currentUser)`. More intuitive to do `parseUser(currentUser)`

Reply:
For simplicity, let's aim to merge this PR first today.